src/nestor/models/notes_book.py

```diff
@@ -1,13 +1,51 @@
 from collections import UserDict
 
+class Note:
+    """Class representing a record for NotesBook."""
+
+    def __init__(self, title, content=None, tags=None):
+        """Initialize a new Note."""
+        self.title = title
+        self.tags = tags if tags else []  # Initialize tags
+        self.note_content = content if content else None  # Initialize note content
```

Comment:
Lets call `note_content` as `body` or `content`. We dont need to add `note` to the property name because it is property of a note already, so to not have cases like `note.note_content` let's rename

Reply:
changed